index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800;900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    
+        <nav>
+        <a id="mylogo" href="#">melbee</a>
+        <div id="humberger-bottun">
+            <span class="ico"></span>
+            <span class="ico"></span>
+            <span class="ico"></span>
+        </div>
+        </nav>
+    
+    
+    <main>
+        <section id="sec">
+        <h1>I’m Melissa
+        Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+        experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#">LET’S CONNECT</a>
+        <div class="img-svg">
+            <img src="/image/Vector.svg" alt="twitter logo">
+            <img src="/image/Linkedin icon.svg" alt=". logo">
+            <img src="/image/Vector (2).svg" alt="N logo">
+            <img src="/image/Vector (3).svg" alt="github logo">
+            <img src="/image/Vector (4).svg" alt="twitter logo">
```

Comment:
- [OPTIONAL] Kindly make sure to use an unordered list to display this list of links to meet the html semantic rules

here is an example:
```
<ul>
<li><a><img/></a></li>
....
</ul>
```